include/linux/slaunch.h

```diff
@@ -66,12 +66,14 @@
 #define TXT_CR_CMD_RESET		0x0038
 #define TXT_CR_CMD_CLOSE_PRIVATE	0x0048
 #define TXT_CR_DIDVID			0x0110
+#define TXT_CR_VER_EMIF			0x0200
```

Comment:
This file is introduced in an earlier patch commit, it would be cleaner to separate this into a separate commit that is tagged to be merged with the patch commit this is introduced in.

Reply:
Yes, these additions should go with commit ba2f29985aba9dc09518a9130c60fc0bc1066b0a